### ISSUE TITLE
Replace R2 score for RMS on cross validation

### ISSUE THREAD

**Issue by santisoler** [compgeolab/eql-gradient-boosted#69]
Following #60, we would need to move from R2 score to RMS also on cross validation of EQL gridders, which is only used on real world cases (Australia gravity for example, on #61).

To be able to use a different score on cross validation we would need to use the latest version of Verde that introduces a new `scoring` argument to [`verde.cross_val_score`](https://www.fatiando.org/verde/dev/api/generated/verde.cross_val_score.html?highlight=cross_val_score#verde.cross_val_score).

I think we can safely pass [sklearn.metrics.mean_squared_error](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html), although new we would set the best predictor as the one that achieves the **minimum** score. Also remember to compute its square root before appending it to the `scores` list.
